src/test/kotlin/blackjack/domain/DeckTest.kt

```diff
@@ -1,21 +1,21 @@
 package blackjack.domain
 
-import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Assertions.assertThrows
 import org.junit.jupiter.api.Test
 
 class DeckTest {
 
     @Test
-    fun `덱이 모두 소멸되면 에러를 던져야 한다`() {
+    fun `덱에 카드가 없으면 에러를 던져야 한다`() {
         // given
-        val deck = Deck()
-        repeat(52) {
-            deck.draw()
-        }
+        val deck = Deck(mutableListOf())
 
-        // when & then
-        assertThatThrownBy { deck.draw() }.isInstanceOf(IllegalStateException::class.java)
-            .hasMessageContaining("덱의 카드들이 모두 사용되었습니다. 다른 덱을 사용해주세요.")
+        // when
+        val result = assertThrows(IllegalStateException::class.java) { deck.draw() }
```

Comment:
> 수현님은 assertThatThrownBy보다 asserThrows를 더 선호하시나요? 처음에 assertThatThrownBy로 배우다가 이번에 asserThrows 알게 되어서 사용해보니 기존 // given, // when, // then 형태를 유지 할 수 있어서 더 가독성이 좋다고 느꼈습니다. 앞으로 asserThrows를 애용하게 될거 같네요 :)

오호 // given, // when, // then 형태를 유지하는 측면에서는 생각해보지 못했는데 감사합니다!

전 개인적으로 
```suggestion
        val result = assertThrows<IllegalStateException> { deck.draw() }
```
요렇게 작성하면 굳이 외부에서 에러 타입에 `::class.java` 을 붙이지 않아도 되어 선호합니다!